src/cpp/ext/otel/otel_plugin.cc

```diff
@@ -80,8 +65,32 @@ absl::flat_hash_set<std::string> BaseMetrics() {
       std::string(grpc::OpenTelemetryPluginBuilder::
                       kServerCallRcvdTotalCompressedMessageSizeInstrumentName)};
 }
+
+// TODO(yashykt): This should go away when we move to the
```

Comment:
Does this mean that the `GlobalStatsPluginRegistry` will have an API to disable metrics recording for a specific (or a group of) `StatsPlugins`? What would that API look like? How can a user specify the `StatsPlugin`s that they want to disable? And would that only affect new channels/servers or existing ones as well?

Reply:
I think the OTelPluginBuilder will probably have new methods - 
1) OTelHandle Build();
2) void Register(OTelHandle);
3) void UnRegister(OTelHandle);

or `BuildAndRegister` returns a handle  which has a method on it to deregister. We discussed that we do not want deregistering to affect existing ones (to avoid synchronization). It'll just affect new ones.